src/components/navbar/RightBlock.tsx

```diff
@@ -105,42 +104,56 @@ const AccountContainer = ({
   )
 }
 
-export default function ({
+const SuspendedAccount = ({
   account,
   needNetworkChange,
   eNSName,
-}: {
-  account?: string
-  needNetworkChange: boolean
-  eNSName?: string
-}) {
+}: AccountProps) => {
   return (
-    <div className={walletContainer}>
-      <SocialLinks />
-      <SealVerse />
-      <LastDelimiter />
+    <Suspense
+      fallback={
+        <div className={accountLinkContainer}>
```

Comment:
We use `accountLinkContainer` twice in this file, this is why I decided not to duplicate it into `NavLoading`